### DISCUSSION TITLE
Lumen compatibility

### DISCUSSION THREAD

**Discussion by itexpertsindya** [ash-jc-allen/short-url#128] [General]
Hi,
Do you know if it works with Lumen?

**Comment by ash-jc-allen** [comment]
Hey @itexpertsindya!

Unfortunately, I don't think this package works with Lumen. A PR was made a while ago to try and add support for it, but I didn't want to merge it in because it meant that I'd have to support Laravel and Lumen. If this is something that enough people want though, it's something that I'd potentially consider adding :)
